TTHAnalysis/python/tools/higgsDiffCompTTH.py

```diff
@@ -165,10 +202,14 @@ def analyze(self, event):
             if len(QFromWFromH)==2:
                 q1, q2 = QFromWFromH
 
-                for x in thejets: # I need these rather than the jetsNoB because I want to allow matching to match the QfromH to b-jets. I need this and not jets[] because I want to access the flavour.
+                # I need these rather than the jetsNoB because I want to allow matching to match the QfromH to b-jets. 
+                # I need this and not jets[] because I want to access the flavour.
+                for _x,x in enumerate(thejets):
+#                    if x.btagDeepB > 0.3093: continue # Optionally kill > Medium B-Jets
                     j=x.p4()
                     j.SetPtEtaPhiM(getattr(x,'pt%s'%jesLabel), j.Eta(), j.Phi(), j.M()) # Correct the pt
-                    jflav = getattr(x,'hadronFlavour')
+#                    jflav = getattr(x,'hadronFlavour')
```

Comment:
The # should be aligned with the other lines at the correct level of indentation (as is done e.g. for L147)